contracts/dmarket.js

```diff
@@ -0,0 +1,1258 @@
+/* eslint-disable no-await-in-loop */
+/* global actions, api */
+
+const UTILITY_TOKEN_SYMBOL = 'BEE';
+
+// 30 days in seconds
+const MAX_EXPIRATION_SECS = 2592000;
+// fee in UTILITY TOKEN required to create a new pair
+const PAIR_CREATION_FEE = '500';
+
+const CONTRACT_NAME = 'dmarket';
+
+actions.createSSC = async () => {
+  const tableExists = await api.db.tableExists('quoteTokens');
+
+  if (tableExists === false) {
+    await api.db.createTable('quoteTokens', ['quoteToken']);
+    await api.db.createTable('buyBook', ['symbol', 'quoteToken', 'account', 'priceDec', 'expiration', 'txId']);
+    await api.db.createTable('sellBook', ['symbol', 'quoteToken', 'account', 'priceDec', 'expiration', 'txId']);
+    await api.db.createTable('tradesHistory', ['symbol', 'quoteToken']);
+    await api.db.createTable('metrics', ['symbol', 'quoteToken']);
```

Comment:
remove indexes (leave as []), and add primaryKey, e.g.:

await api.db.createTable('metrics', [], { primaryKey: ['quoteToken', 'symbol'] });